### DISCUSSION TITLE
我在上周用支付宝续费了一年的蓝灯，但是现在依旧还是将要过期！

### DISCUSSION THREAD

**Discussion by farhigh233** [getlantern/lantern-forum-cn#713] [Q&A]
我在上周用支付宝续费了一年的蓝灯，但是现在依旧还是即将过期！请帮忙检查一下！

我的邮箱 [[231879414@qq.com] @Mike777Khoo

**Comment by MaxMKMax** [comment]
@farhigh233 您好. 请您也在这里提供我支付单据. 谢谢. 我帮您提交问题.

**Reply by farhigh233** [reply]
![微信图片_20240712224131](https://github.com/user-attachments/assets/6f79c122-6593-4fed-8476-e85bd9b714ef)

**Comment by MaxMKMax** [comment]
@farhigh233 好的. 谢谢您. 已经帮您提交问题上去了. 请您给我们几天的时间.

**Comment by henrylyubj** [comment]
请问如何用支付宝支付？
